style.css

```diff
@@ -0,0 +1,79 @@
+body {
```

Comment:
Don't use tag selectors (here and below)

Reply:
Yes, but this project is small, so I decided it would not be a mistake here.